### DISCUSSION TITLE
User Agent Override

### DISCUSSION THREAD

**Discussion by Pidav** [rubycdp/ferrum#351] [Ideas]
I would like to override user agent properly using the following CDP property.
Network.setUserAgentOverride
https://chromedevtools.github.io/devtools-protocol/1-3/Network/#method-setUserAgentOverride
This cdp property is usefull to hide headless status.
We could call it like other cdp with the brower#command method.
Does it seems to be revelant to anyone ?

**Comment by route** [comment]
When you set headers with https://github.com/rubycdp/ferrum#addheaders--boolean we already do override it automatically, take a look at https://github.com/rubycdp/ferrum/blob/13158c019e919f2d0e5f45bb9bb2635b99fe5c64/lib/ferrum/headers.rb#L52
